### PR TITLE
Add completion script for quest Recover the Key.

### PR DESCRIPTION
**Pull request by miraco**
https://github.com/cmangos/tbc-db/issues/847

Based on https://github.com/vmangos/core/commit/7fa3e48bf10b7c40f15b935403e87104453b0b28

"Didn't stay long enough to see how he resets his stand state and npc flags, but he most likely despawns after some time."
Currently he despawns after 2 mins and has a respawn timer from 350seconds. Should recheck this on retail

### CODE REVIEW

File: Updates/0005_Quest_Recover_the_Key.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+DELETE FROM dbscripts_on_quest_end WHERE id = '7846';
+INSERT INTO `dbscripts_on_quest_end` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `data_flags`, `dataint`, `comments`) VALUES 
+('7846', '0', '28', '0', '0', '0', '0', '0', 'Recover the Key!: Elder Torntusk - Stand Up'),
+('7846', '0', '4', '147', '1', '2', '0', '0', 'Recover the Key!: Elder Torntusk - Remove Npc Flags'),
```
Comment (Grz3s) [inline Updates/0005_Quest_Recover_the_Key.sql]: 29 SCRIPT_COMMAND_MODIFY_NPC_FLAGS

File: Updates/0005_Quest_Recover_the_Key.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+DELETE FROM dbscripts_on_quest_end WHERE id = '7846';
+INSERT INTO `dbscripts_on_quest_end` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `data_flags`, `dataint`, `comments`) VALUES 
+('7846', '0', '28', '0', '0', '0', '0', '0', 'Recover the Key!: Elder Torntusk - Stand Up'),
+('7846', '0', '29', '3', '1', '2', '0', '0', 'Recover the Key!: Elder Torntusk - Remove Npc Flags'),
```
Comment (AnonXS) [inline Updates/0005_Quest_Recover_the_Key.sql:4]: command 29 doesnt use datalong3

**Comment by miraco** [comment]
We should recheck this on retail, removing the npcflags and not adding them before he respawns means you cant take the follow quest: return to primal torntusk

Also have to check the quest itself, i get a blue mark if i finish the quest.

**Comment by miraco** [comment]
I still had a boosted character on retail TBC that didnt made this Quest so i checked it myself. 

He dont despawn, and he just removes the Gossip Flag, you can still get the follow up Quest. After ~5 minutes he gets gossip flag back and do an "die" emote to get in old position.

If you want i can upload a video of it.

**Comment by killerwife** [comment]
I actually have a sniff where I stayed around long enough. Will let you know.

**Comment by miraco** [comment]
I updated the script with the information i got from retail and @Grz3s with ActiveObject to be 100% sure that script will go from A to Z. Will recheck it when you found the sniffs.
*Info for myself 5 mins and 8 seconds for last part, event finishs 5mins after Emote OneShotCheer and Text was played. (source retail tbc)

**Pull request merged**